README.md

```diff
@@ -1 +1,64 @@
-# To-Do-List
\ No newline at end of file
+To-Do-List
+
+> This Website adds remove and updatests to-do-list
+
+## Built With
+
+- HTML
+- Javascrict
+
+## Getting Started
+
+To clone this project:
+- Go to github project repository
+- Click on the green button(known as code)
+- Copy the ssh link
+- on your local machine, type the command git clone link
+
+
+### Prerequisites
+- Code editor
+- Knowledge in HTML
+-Knowledge in Javascrict
+-Git
+
+### Setup
+- Clone the repository
+- Change directory into the project folder
+- Install dev dependencies
+- Open project with text editor to view changes
+- Run the code using [No Liveserver]
+
+### Install
+- No Installation is required
+
```

Comment:
- [ ] To install the project, you need to run `npm install`.
- To run the server, you need to run `npm start`. Kindly include this in your `README.md` file.